### ISSUE TITLE
'clone' bug: not all props can be serialized

### ISSUE THREAD

**Issue by necolas** [kodyl/react-document-meta#10]
The library claims to support the following useful pattern (see document-title):

```js
<DocumentMeta>
  <Component />
</DocumentMeta>
```

Otherwise you're required to add a wrapping element and handle any styles it might need to help with layout, which is something I'd like to keep out of containers or routes.

```js
<View>
  <DocumentMeta />
  <Component />
</View>
```

While there is some untested code in place to support this, we (Twitter) encountered a bug that arises from the JSON parse/stringify (your `clone` util) of props that contain unserializable data, e.g., Symbols or functions. I believe the current approach to cloning should collect only the props you're interested in.

**Comment by danieljuhl** [comment]
I'll have a look at this issue, this weekend...

**Comment by danieljuhl** [comment]
Should be fixed in `2.0.1`

**Comment by necolas** [comment]
Thank you!
